nav2_dynamic_params/include/nav2_dynamic_params/dynamic_params_client.hpp

```diff
@@ -175,12 +167,12 @@ class DynamicParamsClient
     return dynamic_param_map_;
   }
 
-  // retreive parameter from map
+  // retreive parameter from map given full path to node
   template<class T>
   bool get_event_param(
```

Comment:
I recommend removing "event" from the name of the various getter functions since getting the values is not only for the event callbacks, but can be done at any time.

Reply:
I would agree. Would it be best to leave as `get_param` and `get_param_or` or to match style with the node and parameter client interface with the full `get_parameter` and `get_parameter_or`?

Reply:
Filed issue #484 .